### ISSUE TITLE
Local remote_url for production files

### ISSUE THREAD

**Issue by sethherr** [bikeindex/bike_index#820]
When working locally with an anonymized copy of the production database (#566), the images from production are referenced in the database.

This means that they don't load, because the remote url for the uploads is local.

Ideally, we'd be able to see images if they were from production, but uploading new images would happen locally (the current development behavior), and those images would point to the correct URL as well.

Basically have our cake and eat it too.
